data.sql

```diff
@@ -27,4 +27,77 @@ VALUES
   5,
   TRUE,
   11
-);
\ No newline at end of file
+);
+
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_in_kg) 
+VALUES 
+(
+  'Charmander',
+  '2020-02-08',
+  0,
+  FALSE,
+  -11
+),
+(
+  'Plantmon',
+  '2022-11-15',
+  2,
+  TRUE,
+  -5.7
+),
+(
+  'Squirtle',
+  '1993-02-04',
+  3,
+  FALSE,
+  -12.13
+),
+(
+  'Angemon',
+  '2005-06-12',
+  1,
+  TRUE,
+  -45
+),
+(
+  'Boarmon',
+  '2005-06-07',
+  7,
+  TRUE,
+  20.4
+),
+(
+  'Blossom',
+  '1998-10-13',
+  3,
+  TRUE,
+  17
+);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
```

Comment:
- [ OPTIONAL] It is not a bad idea to view all the other columns in the table after the update. But I think it would have been nice to only select the column(s) that was updated.

The previous review did give you an example: `SELECT species from animals;`